src/Nethermind/Nethermind.Facade/Filters/NullFilterStore.cs

```diff
@@ -68,6 +68,11 @@ public FilterType GetFilterType(int filterId)
             throw new InvalidOperationException($"{nameof(NullFilterStore)} does not support filter creation");
         }
 
+        public T GetFilter<T>(int filterId) where T : FilterBase
+        {
+            throw new InvalidOperationException($"Filter with ID {filterId} not found");
```

Comment:
Here too.

Reply:
This is consistent with all the other method implementations in class - unless you'd like me to change all of them?

Reply:
Generally, I'd prefer to switching to more concise syntax and file scope namespaces everywhere but it's up to you in this case. PR is approved anyway.